load.php

```diff
@@ -31,8 +32,8 @@
 // Load the Cavalcade Runner CloudWatch extension.
 // This is loaded on the Cavalcade-Runner, not WordPress, crazy I know.
 function boostrap_cavalcade_runner() {
-	// Load the common AWS SDK. bootstrap() is not called in this context.
-	require_once __DIR__ . '/lib/aws-sdk/aws-autoloader.php';
+	// Load the common libraries. bootstrap() is not called in this context.
+	require_once __DIR__ . '/vendor/autoload.php';
```

Comment:
Yeah I think so too, plus an added check for the directory eg `if ( ! class_exists( '\AWS\Whatever' ) && file_exists( __DIR__ . '/vendor/autoload.php' ) )` should be there for anything using `master` branch still.

One other thing is you can configure `composer.json` to autoload some files within the archive too, so we could use that to call `load.php` automatically. Then you'd just need to include `vendor/autoload.php` in `wp-config.php` instead of including `hm-platform/load.php` directly.

@fklein-lu @paulgibbs how do you feel about that approach? Then it can be composer all the way more or less. Will need to do some tests with the installer paths stuff where you send plugins etc to different locations.

Reply:
> Then you'd just need to include vendor/autoload.php in wp-config.php instead of including hm-platform/load.php directly.

True, but I'm not sure about this, as it then depends when the autoload.php is included. hm-platform required being loaded super early (wp-config). At this point I wouldn't jump in to "no need to require hm-platform" yet, I think that is the _goal_, but it's possible some other approach might be the best one, so I'd rather we didn't start changing now how platform is included, just to have to then do it again.

Reply:
Agree the abstraction/hiding of `hm-platform` isn't ideal there...

Although in that instance `hm-platform` could break if the autoloader hadn't been included yet. There would be a change in how teams have to load platform as we'd require `vendor/autoload.php` to be included before `hm-platform/load.php` as some AWS SDK features like XRay might be called too early.

Reply:
That's a good point! I'm still not sure what the ideal bootstrap should be, either way I'm thinking if we go "composer first" for the whole project, really the developer should never have to manually include the autoload file, we should be doing that behind the scened. Equally, there probably shouldn't be a `wp-config.php` -- that's the future where I think we need to get to.